release.sh

```diff
@@ -17,7 +17,7 @@ uhoh() {
 trap uhoh EXIT
 
 STABLE_REGEX="\d+\.\d+\.\d+"
-SNAPSHOT_REGEX="^${STABLE_REGEX}-snapshot\.\d{8}\.\d+(\.\d+)?\.v[0-9a-f]{8}$"
+SNAPSHOT_REGEX="^${STABLE_REGEX}-(snapshot|ancillary)\.\d{8}\.\d+(\.\d+)?\.v[0-9a-f]{8}$"
```

Comment:
Why not using adhoc, similarly to what we are doing in Canton ?

Reply:
Because I forgot; I'll update to `adhoc`.